hack/common.sh

```diff
@@ -6,7 +6,9 @@ KUBEVIRT_ANSIBLE_DIR="$(
 )"
 OUT_DIR=$KUBEVIRT_ANSIBLE_DIR/_out
 TESTS_OUT_DIR=$OUT_DIR/tests
-
+OC_PATH=$(command -v oc)
```

Comment:
These params should be defined in hack/run-tests.sh ..
if we leave it here it will enforce user to have virtctl be installed even for compilation ... which is not good thing in general ...